### PR TITLE
Add 5 questions to rust-quiz.md

### PR DESCRIPTION
**Pull request by ahr9n**

### CODE REVIEW

**Review by ahr9n — COMMENTED** [review]
What is a difference between **\_** and **\_______** in markdown?

File: rust/rust-quiz.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ str::thread::spawn(|| {
 - [ ] boolean
 - [ ] tuple
 
-#### Q4. **\_** cannot be destructured.
+#### Q4. **\_______** cannot be destructured.
```
Comment (Ebazhanov) [inline rust/rust-quiz.md:30]: this one will be updated by prettier, please don't spend time on it

File: rust/rust-quiz.md [hunk 2]
```diff
@@ -40,3 +40,49 @@ str::thread::spawn(|| {
...
+
+```rust
+fn increment(i: T) {
+  // body elided
+}
+```
+- [ ] *mut T
+- [ ] mut ref T
+- [ ] mut &T
+- [ ] &mut T
```
Comment (Ebazhanov) [inline rust/rust-quiz.md:88]: thanks for the new questions!!!!

**Review by Ebazhanov — APPROVED** [review]
LGTM 👍

**Pull request merged**